extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -66,6 +68,9 @@
     @Inject
     BuildProducer<ReflectiveHierarchyBuildItem> reflectiveHierarchyClass;
 
+    @Inject
+    BuildProducer<ReflectiveMethodBuildItem> reflectiveMethod;
```

Comment:
Man do I dislike this pattern...

But of course I also like consistency, so better be consistently ugly :)

Reply:
Yeah, same here. I could change it but we have more pressing things to do, I think :).

Reply:
Absolutely!